### ISSUE TITLE
InstallSchema error or foget to remove comment

### ISSUE THREAD

**Issue by giuseppemorelli** [magespecialist/m2-MSP_CashOnDelivery#1]
Hi guys,
after install your extension and try to enter in the payment section on magento admin there is this error:

```
main.CRITICAL: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'local_magento2.msp_cashondelivery_table' doesn't exist, query was: SELECT `msp_cashondelivery_table`.* FROM `msp_cashondelivery_table` [] []
```

After some check I found that in your InstallSchema.php file there is the function that create the table commented.

https://github.com/magespecialist/m2-MSP_CashOnDlivery/blob/master/Setup/InstallSchema.php

```
    /**
     * {@inheritdoc}
     */
    public function install(SchemaSetupInterface $setup, ModuleContextInterface $context)
    {
        $setup->startSetup();
        //$this->_setupTable($setup, $context);
        $setup->endSetup();
    }
```

If I uncomment the line, the script run correctly and the table will be create.
Everything is fine after that.

Please, can you fix that?
Thanks

Ps. maybe I can send a Pull Request if you want.

**Comment by midimarcus** [comment]
Thank you Giuseppe.
I've assigned this issue, in case there is already a new version ready to upload without pull request.

**Comment by midimarcus** [comment]
Moved to @giacmir for fixing.

**Comment by phoenix128** [comment]
@giuseppemorelli, thank to @giacmir job it should be fixed in 0.2.0. Please upgrade.
Thank you ;)
